packages/server/tests/acceptance/estimateGasPrecompile.spec.ts

```diff
@@ -665,6 +992,99 @@ describe('EstimatePrecompileContract tests', function () {
     isWithinDeviation(gasResult.gasUsed, estimateGasResponse, lowerPercentBound, upperPercentBound);
   });
 
+  //EGP-035
+  it('should call estimateGas with cryptoTransfer function for hbars', async function () {
+    const cryptoTransfers = {
```

Comment:
Can we extract this function and make it with parameter for the amount, because I can see that we are using it in other tests as well.

Reply:
Which function exactly you are referring to? on row 997 its a constant with parameters for the transfer